config/config.yaml

```diff
@@ -12,6 +12,8 @@ polygon: [[0.5, -0.5],[0.5, 0.5], [-0.5, 0.5], [-0.5, -0.5]] #in float
 
 polygon_topic_name: "/lidar_crop_polygon" #topic name of polygon
 
+angle_min: [1.570796,1.570796] #in double
+angle_max: [4.71239,4.71239] #in double
 
```

Comment:
further in the scan ( line 1) you are taking in only 1 input scan. shouldn't you be taking input from atleast 2 scans? since you are providing two angle min and angle max